--- test-vm1.yaml	2023-09-05 22:11:33.994124688 +0200
+++ test-vm1.yaml.good	2023-09-05 21:51:18.000000000 +0200
@@ -1,6 +1,6 @@
 ---
 - name: "vm1"
-  node_name: "eva"
+  node_name: "pve5"
   resource_name: "vm1"
   vm_id: 1000
   agent:
@@ -13,14 +13,14 @@
     sockets: 1
   cloud_init:
     type: "nocloud"
-    datastore_id: "local-lvm"
+    datastore_id: "local"
     dns:
-      domain: "homelab.lan"
+      domain: "iutbeziers.fr"
       server: "1.1.1.1 1.0.0.1"
     ip_configs:
       - ipv4:
-          address: "192.168.0.110/24"
-          gateway: "192.168.0.254"
+          address: "192.168.2.110/24"
+          gateway: "192.168.2.254"
       # - ipv6:
       #     address: "fd91:0812:a17f:6194::10/64"
       #     gateway: "fd91:0812:a17f:6194::1"
@@ -28,21 +28,21 @@
       username: "ubuntu"
       password: "ubuntu"
       keys:
-       - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC2j0Hf+W51tQBTwekIf1lQdEX1pNfY5TXw35P02nBHAqH69WzQpPJTuaYs+WmPbZ/066AssABYtE8JgR2gUzAWL010Yg1iNERPxy41UtFIF1HSV7wGi3kMruBzFCCbEBvbRFkJc1vQv4tS7G0UPtLiTHLYadsHq27A0lZJ/y/TsDvDTY0olbkfit3juMExy5uzCJN5Db1yiRNtBkl8vZP183kLatGtGygz1o9G6ywqZ51Tl2sgYH1ahCheuTv6Zn82B2jzrvisrza7hO0goEg2WR7SWbVl3AnA4YncJ5x9L8PXifXVCFVUj7p8PXqcu7KOixp9IGANvtiTfdvzm4Xg3BLRBKNJXWXW1f19MtPzOUMuRWru4hj6kX20/3C8WB8yco+qQlI0I6+5QYiGQtl4ZHsXcrCTyKtPmYwCdHv9XcEdDGXVNY/4oLm3rh06Nf8ZNx3phZj3xfZ8Lx+BohRRanVJifyKjucopcSy75b9lEMeWKlySLlTQjXDbBtPzSfupVE2Gvw3969J7uAIvvzASdEKdNgTF/ZEUnLexVI82HXpo2GHjr9Fj2B0JhSBfttsyd5LvubbuJ1HVOtPrax971vsUENoUuRLbJPbLoFdCoE5CKKReyjAYIbE9oDEIVt0KlnHI+gAsYg078uwyxrD8nallrBIx+eOG5hDmnum8w== jbriault@deezer.com
+       - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCluScPESl1bpumb6lFN/0KEKhEcI+W0VnttVz2jyh9lbwyMoo6eZA7QbwhXx3Qy85u1tD9G1Q6ayzaJ0KdWTRVgqpNA+hWDZ09913zgQS3vnXPvNIBHqZkapnQE7GbegsbCvwfX6ePNBMem9CE8aOKy/6hJU6/ncn1Sf7NxcElRt4xFk9/D0zEjruwApZaWYAT1/O54hfMn1kdkYrRSrz33inR7NPO/LhcJ1WKe+OmDowaEANO49Yr0y83ZzQOmvhBJSfzVA5w2tQIImmWxOagDVEe1kXlyEYqezYcyF77sEAxJTjY90KYOf6DMwn8ke5A0wLDc70cLNR9/UnDJBGr pouchou@portablejmp
   clone:
-    node_name: "eva"
+    node_name: "pve5"
     vm_id: 9000
     full: true
   disks:
     - disk1:
         interface: "scsi0"
-        datastore_id: "local-lvm"
+        datastore_id: "local"
         size: 32
         file_format: "raw"
         cache: "none"
     - disk2:
         interface: "scsi1"
-        datastore_id: "local-lvm"
+        datastore_id: "local"
         size: 32
         file_format: "raw"
         cache: "none"
@@ -50,14 +50,14 @@
     dedicated: 512
   network_devices:
     - net1:
-        bridge: "vmbr0"
+        bridge: "vmbr1"
         model: "virtio"
     - net2:
-        bridge: "vmbr0"
+        bridge: "vmbr1"
         model: "virtio"
   on_boot: true
 - name: "vm2"
-  node_name: "eva"
+  node_name: "pve5"
   resource_name: "vm2"
   vm_id: 1001
   agent:
@@ -70,14 +70,14 @@
     sockets: 1
   cloud_init:
     type: "nocloud"
-    datastore_id: "local-lvm"
+    datastore_id: "local"
     dns:
-      domain: "homelab.lan"
+      domain: "iutbeziers.fr"
       server: "1.1.1.1 1.0.0.1"
     ip_configs:
       - ipv4:
-          address: "192.168.0.111/24"
-          gateway: "192.168.0.254"
+          address: "192.168.2.111/24"
+          gateway: "192.168.2.254"
       # - ipv6:
       #     address: "fd91:0812:a17f:6194::10/64"
       #     gateway: "fd91:0812:a17f:6194::1"
@@ -85,21 +85,21 @@
       username: "ubuntu"
       password: "ubuntu"
       keys:
-       - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC2j0Hf+W51tQBTwekIf1lQdEX1pNfY5TXw35P02nBHAqH69WzQpPJTuaYs+WmPbZ/066AssABYtE8JgR2gUzAWL010Yg1iNERPxy41UtFIF1HSV7wGi3kMruBzFCCbEBvbRFkJc1vQv4tS7G0UPtLiTHLYadsHq27A0lZJ/y/TsDvDTY0olbkfit3juMExy5uzCJN5Db1yiRNtBkl8vZP183kLatGtGygz1o9G6ywqZ51Tl2sgYH1ahCheuTv6Zn82B2jzrvisrza7hO0goEg2WR7SWbVl3AnA4YncJ5x9L8PXifXVCFVUj7p8PXqcu7KOixp9IGANvtiTfdvzm4Xg3BLRBKNJXWXW1f19MtPzOUMuRWru4hj6kX20/3C8WB8yco+qQlI0I6+5QYiGQtl4ZHsXcrCTyKtPmYwCdHv9XcEdDGXVNY/4oLm3rh06Nf8ZNx3phZj3xfZ8Lx+BohRRanVJifyKjucopcSy75b9lEMeWKlySLlTQjXDbBtPzSfupVE2Gvw3969J7uAIvvzASdEKdNgTF/ZEUnLexVI82HXpo2GHjr9Fj2B0JhSBfttsyd5LvubbuJ1HVOtPrax971vsUENoUuRLbJPbLoFdCoE5CKKReyjAYIbE9oDEIVt0KlnHI+gAsYg078uwyxrD8nallrBIx+eOG5hDmnum8w== jbriault@deezer.com
+       - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCluScPESl1bpumb6lFN/0KEKhEcI+W0VnttVz2jyh9lbwyMoo6eZA7QbwhXx3Qy85u1tD9G1Q6ayzaJ0KdWTRVgqpNA+hWDZ09913zgQS3vnXPvNIBHqZkapnQE7GbegsbCvwfX6ePNBMem9CE8aOKy/6hJU6/ncn1Sf7NxcElRt4xFk9/D0zEjruwApZaWYAT1/O54hfMn1kdkYrRSrz33inR7NPO/LhcJ1WKe+OmDowaEANO49Yr0y83ZzQOmvhBJSfzVA5w2tQIImmWxOagDVEe1kXlyEYqezYcyF77sEAxJTjY90KYOf6DMwn8ke5A0wLDc70cLNR9/UnDJBGr pouchou@portablejmp
   clone:
-    node_name: "eva"
+    node_name: "pve5"
     vm_id: 9000
     full: true
   disks:
     - disk1:
         interface: "scsi0"
-        datastore_id: "local-lvm"
+        datastore_id: "local"
         size: 32
         file_format: "raw"
         cache: "none"
     - disk2:
         interface: "scsi1"
-        datastore_id: "local-lvm"
+        datastore_id: "local"
         size: 32
         file_format: "raw"
         cache: "none"
@@ -107,9 +107,9 @@
     dedicated: 512
   network_devices:
     - net1:
-        bridge: "vmbr0"
+        bridge: "vmbr1"
         model: "virtio"
     - net2:
-        bridge: "vmbr0"
+        bridge: "vmbr1"
         model: "virtio"
   on_boot: true
